src/FormBuilder/ServiceProvider.php

```diff
@@ -46,5 +46,12 @@ public function boot()
                 'newFormUrl' => FormBuilderRouteBuilder::makeCreateFormRoute()->getUrl(),
             ]);
         });
+        
+        /** Dynamically renders Next Gen Forms using the Next Gen Form block. */
+        add_action('give_before_single_form', function() {
```

Comment:
This is a work in progress. I ran into some issue is replacing the form content and will need a separate PR to really address how this should work. But I wanted to take a step in that direction and not further convolute this change.

Reply:
We're going to need to override the single give form completely.  Not sure if thats possible via a plugin but maybe we can do `template_include` to replace that file with ours.

Reply:
I just tested this out and it works:

```
 add_filter('template_include', static function ($template) {
      if (is_single() && get_post_type() === 'give_forms') {
          $template = GIVE_NEXT_GEN_DIR . 'templates/givewp-single-form.php';
      }

      return $template;
  }, 99);
```